### DISCUSSION TITLE
How to dynamically update swiper properties after it has been created

### DISCUSSION THREAD

**Discussion by ajssd** [nolimits4web/swiper#5175] [Q&A] [answered]
I could not find anything in the documentation about how to **modify the properties of an existing swiper instance** after it has been instantiated.  Is it possible?

For instance, let's say I start with this:
```
const swiper = new Swiper('.swiper', {
  speed: 400
});
```
and then I have some hypothetical UI that slows it down:
```
const slowModeClicked = function(event) {
  const swiper = document.querySelector('.swiper').swiper;
  swiper.speed = 1000;  // HOW DO I GET THIS TO STICK?
}
```
I tried adding a call to `swiper.update()` but that didn't work either.  Is it supported?

**Comment by nolimits4web** [comment, marked as answer]
`swiper.params.speed = 1000;`

**Reply by reza-ghafoori** [reply]
how to do this on a react component, please?
